intials.c

```diff
@@ -0,0 +1,18 @@
+/* Matthew Danielson
```

Comment:
Perfect code! 💯 You followed all the guidelines and your code runs! Great job! 😄